docs/lesson1.md

```diff
@@ -1,33 +1,32 @@
-Here's a markdown outline/template you can use for the hands-on labs developed for your Developer Productivity training. This structure ensures clarity, consistency, and ease of use for participants.
-
----
 
 # Hands-On Lab: [Installing and Configuring GitHub Copilot]
 
 ## Overview
 
 **Goal:**  
-Provide a brief description of the lab's purpose. Explain what participants will learn or accomplish by the end of the session.
+In this lab, you will learn how to install:
+- GitHub Copilot extension in Visual Studio Code
+- GitHub Copilot CLI
 
 **Estimated Duration:**  
-[Time in minutes/hours]
+[10-15 minutes]
 
 **Audience:**  
-[Target audience, e.g., developers, engineers, etc.]
+[developers]
```

Comment:
i think we should add testers and ops at a minimum

Reply:
added